pkg/metadata/main.go

```diff
@@ -167,25 +167,31 @@ func processUserData(data []byte) error {
 			continue
 		}
 		for f, i := range files {
-			fi := i.(map[string]interface{})
-			if !ok {
-				log.Printf("Could convert JSON for items: %s", i)
-				continue
-			}
-			if _, ok := fi["perm"]; !ok {
-				log.Printf("No permission provided %s:%s", f, fi)
-				continue
-			}
-			if _, ok := fi["content"]; !ok {
-				log.Printf("No content provided %s:%s", f, fi)
-				continue
-			}
-			c := fi["content"].(string)
-			p, err := strconv.ParseUint(fi["perm"].(string), 8, 32)
-			if err != nil {
-				log.Printf("Failed to parse permission %s: %s", fi, err)
+			p := uint64(0666)
+			var c string
+
+			switch fi := i.(type) {
+			case map[string]interface{}:
+				if _, ok := fi["perm"]; !ok {
+					log.Printf("No permission provided %s:%s", f, fi)
+					continue
+				}
+				if _, ok := fi["content"]; !ok {
+					log.Printf("No content provided %s:%s", f, fi)
+					continue
+				}
+				c = fi["content"].(string)
+				if p, err = strconv.ParseUint(fi["perm"].(string), 8, 32); err != nil {
```

Comment:
Yeah, if err is set it'll hit the `continue` and never look at `p` and skip the file (like it did before)